Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aya-rustc-llvm-proxy"
-version = "0.3.0"
+version = "0.3.1"
```

Comment:
please don't change this or `cargo-release` will get confused

Reply:
changed back to "0.3.0"